modelskill/metrics.py

```diff
@@ -678,3 +699,162 @@ def add_metric(
 
     # add the function to the module
     setattr(sys.modules[__name__], metric.__name__, metric)
+
+
+def pr(obs: np.ndarray, model: np.ndarray) -> float:
+    """PR: Peak Ratio
+
+    PR is the ratio of the mean of the identified peaks in the model / identified peaks in the measurements
+
+    .. math::
+            \\frac{\\sum_{i=1}^{N_{peak}} (model_i)}{\\sum_{i=1}^{N_{peak}} (obs_i)}
+
+    Range: [0, inf]; Best: 1.0
+
+    """
+
+    assert obs.size == model.size
+    if len(obs) == 0:
+        return np.nan
+    time=obs.index
```

Comment:
I think we should have an assert statement before this line checking that the inputs are indeed dataframes with a DatetimeIndex. All other metrics just assumes input is array-like.

Reply:
Actually.... we changed that in a previous PR, so the metrics inputs are now Pandas.Series I understand
https://github.com/DHI/modelskill/pull/215
I just copied the same lines from all the other metrics. But from the previous pull request I understand that now that the inputs changed to pandas series instead of arrays :S maybe be should change the all the metrics from 
`obs:array` to `obs:series`

what do you say? Maybe I am wrong and inputs are still arrays.